.github/ISSUE_TEMPLATE/add_project.md

```diff
@@ -6,19 +6,30 @@ labels: add-project
 assignees: ''
 ---
 
+*Please fill up the following template as thoroughly as possible.
+This will help other members better understand the request and speed up the decision making process.*
```

Comment:
Could you turn this into a comment so that we don't have to delete it to create a clean request?
```suggestion
<!--
Please fill up the following template as thoroughly as possible.
This will help other members better understand the request and speed up the decision making process.
-->
```